src/System.IO.FileSystem/src/System/IO/Win32FileStream.cs

```diff
@@ -1251,6 +1276,11 @@ public override int ReadByte()
             return result;
         }
 
+        private bool ActiveBufferOperation()
```

Comment:
I think this would be better as a property, e.g.
```C#
private bool HasActiveBufferOperation
{
    get { return _activeBufferOperation != null && !_activeBufferOperation.IsCompleted; }
}
```